memberships/templates/base.html

```diff
@@ -66,8 +69,8 @@ <h1><a href="/" title="Home">Geek.Zone</a></h1>
     </main>
 
     <footer>
-      <h3><span style="display: inline-block; transform: rotateY(180deg);">&copy;</span> Geek.Zone</h3>
-      <div><button class="button" onclick="toggleDarkMode()">Toggle Dark Mode</button></div>
+      <h3 class="copyright"> &copy; Geek.Zone</h3>
```

Comment:
I wonder if it is semantically correct to have this as an h3 tag? Since this is not strictly a header.

If you imagine using a screen reader or some software that parses the content of a website into a hierarchy it would think this was a subtitle of an article or something.

I'm not sure what would be the correct tag to use instead, maybe just a `<span>` or a `<p>` tag?

Reply:
I think theres a small tag used for the copyright which i’ll incorporate but if not will use the p tag.